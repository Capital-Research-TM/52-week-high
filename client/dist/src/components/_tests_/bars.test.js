import React from 'react';
import Bars from '../bars.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Bars />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});