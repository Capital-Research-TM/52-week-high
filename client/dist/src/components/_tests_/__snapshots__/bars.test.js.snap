// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="containGraphToBottomBar"
>
  <div />
  <div
    className="barContainer"
  />
  <div
    className="overlay"
  >
     
  </div>
  <div
    className="fiftyTwoWeek"
  >
    <div
      className="lowest"
    >
      <span>
        52 week low 
      </span>
      <span>
        0.00
      </span>
    </div>
    <div
      className="highest"
    >
      <span>
        52 week high
      </span>
      <span>
        100.00
      </span>
    </div>
  </div>
</div>
`;
